packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.jsx

```diff
@@ -1,31 +1,34 @@
 import React, { useState } from 'react';
-import Button from 'terra-button';
-import Toggle from 'terra-toggle';
+import ShowHide from 'terra-show-hide';
 import Alert from 'terra-alert';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
```

Comment:
Duplicative import with previous line, can remove line 5 and update line 9 to just:
```
const cx = classNames.bind(styles);
```